examples/08_experimental/plot_surface_image_and_maskers.py

```diff
@@ -52,8 +53,8 @@
 for view, ax_row in zip(views, axes):
     for ax, hemi in zip(ax_row, hemispheres):
         plotting.plot_surf(
-            mean_img,
-            part=hemi,
+            surf_map=mean_img,
+            hemi=hemi,
             view=view,
             figure=fig,
             axes=ax,
```

Comment:
I think that this figure would render better if se use 'bg_on_data=True' ? Alternatively, we could also threshold the image.

Reply:
done but the new plot does not look tons better